README.md

```diff
@@ -73,112 +72,120 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
-- Improved performance when loading a big number of files.
-- Fixed a specific case in which PowerRename tried to rename a file to an empty string.
-- The UI now shows when a file can't be renamed due to its name being too long or containing invalid characters.
+- Fixed a bug causing File Explorer to crash under some conditions when accessing the context menu.
 
 ### PowerToys Run
 
-- Added a fix to the VSCodeWorkspaces plugin to better support portable installations. Thanks [@bkmeneguello](https://github.com/bkmeneguello)!
-- The Folder plugin now expands `%HOMEPATH%` correctly.
-- Fixed a case where a previous result was being activated when searching for new results. Added a setting to better control input throttling. Thanks [@jefflord](https://github.com/jefflord)!
-- Added support for port numbers in the URI plugin. Thanks [@KohGeek](https://github.com/KohGeek)!
-- Fixed query errors when the search delay option was turned off.
-- New History plugin to search for old search results. Thanks [@jefflord](https://github.com/jefflord)!
-- Changed the default TimeDate activation keyword to `)`, as queries starting by `(` are expected as Calculator global queries, and added information in Settings so users know that some activation keywords may conflict with normal usage of some plugins when trying to do a global query. Thanks [@htcfreek](https://github.com/htcfreek)!
-- The Unit Converter plugin updated its UnitsNet dependency and now supports plural units. Thanks [@FWest98](https://github.com/FWest98)!
-- Improved the validation logic in the Calculator plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Added support to opening Terminal windows in quake mode. Thanks [@FWest98](https://github.com/FWest98)!
+- Fixed utility exit logic to close all threads and avoid crashes. (This was a hotfix for 0.62)
+- Improve the icon shown in the Program plugin for application execution aliases. Thanks [@MikeBarker-MSFT](https://github.com/MikeBarker-MSFT)!
+- Fix calls to the default browser when Firefox is installed from the Microsoft Store.
+- Fixed accessibility issue in which controls appended to the result entries weren't announced.
+- Search was improved and should now return results where the terms in the query appear at the end of the result.
+
+### Quick Accent
+
+- Improved the keyboard hooks performance. (This was a hotfix for 0.62)
+- Fixed a bug that was causing Quick Accent to interfere with Keyboard Manager. (This was a hotfix for 0.62)
+- Added the correct ß uppercase character. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+- Accent character selection should now wrap around. Thanks [@wmentha](https://github.com/wmentha)!
+- Added language selection setting to reduce the number of accented characters shown. The available languages are Currency, Czech, Dutch, French, Hungarian, Icelandic, Italian, Maori, Pinyin, Polish, Romanian, Slovakian, Spanish and Turkish. Thanks [@damienleroy](https://github.com/damienleroy)!
```

Comment:
Germany is missing. Did you write Dutch instead of Germany?

Reply:
Sharp one.
Also: [Pinyin](https://wikipedia.org/wiki/Pinyin) is not a **language**. It's a way to transliterate Chinese (possibly Taiwanese) to roman alphabet.
@damienleroy I think it would be even better to name these _subsets_ in stead of languages. Also looking at "Currency"...

Reply:
I see no German, there 🤷  
![image](https://user-images.githubusercontent.com/26118718/193681629-57c68ba9-c9f1-4a36-93c0-38d9299c1d3b.png)

Reply:
Guess it might be better to just remove the language list from the notes at this time?

Reply:
Or just call them character sets?

Reply:
Changed it to "The available accented character sets are ..."

Reply:
Have a look over at Damien's repo and there we see German (DE) and no Dutch (NL). Must have gone lost in translation, so to say 😅